
var catchError = function(err) {
	console.error(err);
	
}

module.exports = catchError;
